### PR TITLE
Extract metric from a test for benchmarkAI

### PR DESCRIPTION
**Pull request by karan6181**
- Save metrics from a test for benchmarkAI.

@sandeep-krishnamurthy @roywei @kalyc

### CODE REVIEW

File: benchmark/scripts/logging_metrics.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+        if hasattr(time_callback, 'times'):
+            self.metric_list.append(time_callback.get_time())
+            self.metric_list.append(time_callback.get_time_step())
+            self.metric_list.append(time_callback.get_speed())
+            self.pattern_list.append('time: %s\t')
+            self.pattern_list.append('time_step: %s\t')
+            self.pattern_list.append('speed: %s\t')
+
+        if 'loss' in history_callback.history:
```
Comment (kalyc) [inline benchmark/scripts/logging_metrics.py]: please move the metric constants to constant.py file & iterate over those to increase modularity

File: benchmark/scripts/models/timehistory.py [hunk 2]
```diff
@@ -8,11 +8,69 @@
...
+
+    def get_num_samples(self):
+        if 'samples' in self.params:
+            return self.params['samples']
+        elif 'steps' in self.params:
+            return self.params['steps']
+        else:
+            raise ValueError('Incorrect metric parameter')
+
+    def reformat(self, var):
```
Comment (kalyc) [inline benchmark/scripts/models/timehistory.py]: please make this a private method & limit its scope or move it to utils such that LoggingMetrics class can take advantage of it while appending to pattern list

**Comment by sandeep-krishnamurthy** [comment]
As discussed offline, these custom BenchmarkAI specific changes should not be made here in Keras repo.

**Comment by karan6181** [comment]
This PR code change is old. Created a new PR which has all the latest changes ( https://github.com/awslabs/keras-apache-mxnet/pull/78)

**Comment by roywei** [comment]
@karan6181 closing this PR as it's out dated

**Pull request closed without merging**